src/web/modules/gateway_accounts/gateway_accounts.http.ts

```diff
@@ -166,13 +166,35 @@ const writeAccount = async function writeAccount(req: Request, res: Response): P
     }
   }
 
+  const stripeStatementDetails: {[key: string]: string} = {
+    payoutStatementDescriptor: '',
+    statementDescriptor: ''
+  } 
+
+  if (account.provider === 'stripe'){
+    const stripeAccountDetails = await stripe.accounts.retrieve(account.credentials);
+    stripeStatementDetails.payoutStatementDescriptor =  stripeAccountDetails.payout_statement_descriptor
+    stripeStatementDetails.statementDescriptor =  stripeAccountDetails.statement_descriptor
+  }
+
+  const baseGoLiveUrlMap: {[key: string]: string} = {
+    'test-12': 'https://selfservice.pymnt.uk/',
+    'staging-2': 'https://selfservice.staging.payments.service.gov.uk/',
+    'production-2': 'https://selfservice.payments.service.gov.uk/'
+  }
```

Comment:
This should use a SELFSERVICE_URL environment variable like we do for other URLs

This will need to be set in https://github.com/alphagov/pay-infra/blob/master/provisioning/terraform/modules/pay_microservices/toolbox/toolbox.tf#L71

Reply:
Good idea - this has now been done.